src/config.ts

```diff
@@ -33,6 +33,13 @@ export const cliOptions: IOptionConfig<ICLIOptions>[] = [
     request: "Please specify RPC endpoint.",
     default: defaultRPC,
   },
+  {
+    letter: "i",
```

Comment:
1. So unfortunate that ‘p’ is occupied by the priority param, ‘i’ not so obsious. Do we want to drop a line about it in the root readme?